srtcore/srt.h

```diff
@@ -562,6 +573,7 @@ enum SRT_REJECT_REASON
 #ifdef ENABLE_AEAD_API_PREVIEW
     SRT_REJ_CRYPTO,      // conflicting cryptographic configurations
 #endif
+    SRT_REJ_SETTINGS,    // socket settings on both sides collide and can't be negotiated
```

Comment:
We already use the "config" naming throughout the project, e.g. socket config. Another naming is "options", e.g. socket option. I suggest using one of those to avoid introducing yet another term.

Reply:
Ok, I think you're right. And I've seen one more trick - the numbers should be explicitly assigned, if there is a conditional in the API, in order to avoid ABI compat tricks.